clientlibs/js/webpack.config.ts

```diff
@@ -34,8 +34,10 @@ const browser = {
   output: {
     filename: 'index.js',
     path: path.resolve(__dirname, 'dist/browser'),
+    libraryExport: 'default',
```

Comment:
There are four variations where the same change needs to be done. Can you refactor the code and extract the common code?